releasenotes/notes/fix-adjvector-to-file-8e56711f5d4a4233.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Fix (deprecated) ``AdjVectorBQM.to_file`` and ``AdjVectorBQM.from_file``.
+    See `#898 <https://github.com/dwavesystems/dimod/issues/898>`_.
```

Comment:
@randomir , as a formatting thing, do we want this kind of format, or more explicit like `GitHub dwavesystems/dimod#898`, or use a format [like numpy](https://numpy.org/doc/stable/release/1.21.0-notes.html) e.g. `gh-898`.

Reply:
I would keep it simple with `#898` only. Unless we want to keep option of leaving GH open...

Reply:
Simple for the win!